package/MDAnalysis/core/topologyobjects.py

```diff
@@ -116,13 +118,12 @@ def __hash__(self):
         return hash((self._u, tuple(self.indices)))
 
     def __repr__(self):
-        indices = (self.indices if self.indices[0] < self.indices[-1]
-                   else self.indices[::-1])
+        "Return representation in same order of AtomGroup indices"
```

Comment:
```suggestion
        """Return representation in same order of AtomGroup indices"""
```
It's more common for docstring to use the triple " convention.